Sources/ARKit-CoreLocation/Extensions/SCNNode+Extensions.swift

```diff
@@ -12,6 +12,7 @@ extension SCNNode {
     /// Overlapping nodes require unique renderingOrder values to avoid flicker
     /// This method will select random values if you don't care which node is in front of the other,
     /// or you can specify a particular z-order value
+	// Note: rendering order will be changed later based on distance from camera
```

Comment:
Please fix the spacing here and use three slashes: `///` so that the comments get picked up by Xcode

Reply:
I'm happy to fix, but don't understand "so that the comments get picked up by Xcode"

Reply:
In Xcode, when you "option-click" on a function or property or class, it will show you the documentation for that function or property or class by reading comments that have 3 slashes (2 slashes are ignored - and having a 2 slash comment after the 3 slash comment may actually break the entire function comment with respect to Xcode's ability to read them).

Reply:
I had no idea of that feature, thanks.

Reply:
My pleasure!  Thanks for being open to the feedback.  This is another great feature, I'm stoked to get this merged in.